getting_started/setup_linux/install_locally.rst

```diff
@@ -0,0 +1,106 @@
+.. _install-locally:
+
+Install the Kernel locally
```

Comment:
There is already a gazillion of guides for this on the Internet, could you add a couple pointers to the best ones? As in, "for further details..."

Reply:
What would the benefit of that be? The included instructions cover the basics, and people who want to do more advanced stuff would already be familiar with kernel development. It seems like all those links would do is confuse people as they wouldn't know what to follow.